kubernetes/helm/collabora-online/templates/configmap.yaml

```diff
@@ -12,9 +12,9 @@ data:
   {{- end }}
   server_name: {{ .Values.collabora.server_name }}
   {{- range $k,$v := .Values.collabora.aliasgroups }}
-  {{- $alias := $v.domain }}
+  {{- $alias := $v.host }}
```

Comment:
This is a breaking change - was it necessary?

Reply:
Not a necessary but. Wanted to be consistent with how alias_groups are defined everywhere . Sure if it breaks let me change it

Reply:
https://github.com/CollaboraOnline/online/pull/5632

Reply:
it is okay - i was just wondering ...

Reply:
@genofire should I revert the change or no ?